src/repository/logic.py

```diff
@@ -367,7 +367,7 @@ def get_published_preprints(request):
         'preprintauthor_set'
     )
 
-    if request.user.is_staff:
+    if request.user.is_staff or request.user in request.repository.managers.all():
```

Comment:
I think would be nice to use the function you just added above:
`if request.user.is_staff or request.user.is_repository_manager(request.repository)`